internal/web/app.go

```diff
@@ -125,6 +125,10 @@ func buildHandler(c echo.Context) error {
 	}
 
 	variant := c.FormValue("variant")
+	architecture := c.FormValue("architecture")
+	if architecture == "" {
+		architecture = "amd64" // Default to amd64 if not specified
```

Comment:
can we somehow default to host? or by default does it have xcompilation?

Reply:
The architecture of the machine running the build too (auroraboot), doesn't necessarily have anything to do with the machine that is going to run Kairos. Maybe a default value is wrong and we should always require and explicit value from the form. What do you think?

Reply:
done. Made it a mandatory param